core/utils.go

```diff
@@ -301,3 +303,37 @@ func FileExists(path string) (bool, error) {
 
 	return false, err
 }
+
+// get body text with a text editor
+// works with terminal based editors
+func getBodyTextWithEditor() (string, error) {
+	bodyTxtFile, err := os.CreateTemp(os.TempDir(), "changie-body-txt-")
+	if err != nil {
+		return "", err
+	}
+	defer bodyTxtFile.Close()
+	defer os.Remove(bodyTxtFile.Name())
```

Comment:
I think these need to be in the other order, defer uses a stack so we would delete the file and then try to close it.